conf/gulp-tasks/library.gulpfile.js

```diff
@@ -68,7 +68,8 @@ function legacyBundleIIFE () {
   return legacyBundle({
     format: 'iife',
     name: NAMESPACE,
-    sourcemap: true
+    sourcemap: true,
```

Comment:
Do we need to fix this for the modern bundle as well?

Reply:
nope! just checked, it's not an issue for the modern bundle. I believe it's because the modern bundle doesn't have the regenerator-runtime polyfill (or any of the polyfill-prefix stuff)?

Reply:
updated the test desc.